plugins/newsdownloader.koplugin/main.lua

```diff
@@ -276,6 +285,17 @@ function NewsDownloader:downloadFeed(feed, feed_output_dir)
     httpRequest({ url = link, sink = ltn12.sink.file(io.open(news_dl_path, 'w')), })
 end
 
+function NewsDownloader:createFromDescription(feed, feed_output_dir)
+    local news_file_path = ("%s%s%s"):format(feed_output_dir,
+                                           util.replaceInvalidChars(getFeedTitle(feed.title)),
+                                           file_extension)
+    logger.dbg("NewsDownloader: News file will be created :", news_file_path)
+    local io = require("io")
+    local file = io.open(news_file_path, "w")
+    file:write("<!DOCTYPE html><html><head><meta charset='UTF-8'></head><body><header><h2>".. feed.title .. "</h2></header><article>" ..feed.description .."</article></body></html>")
```

Comment:
It's not the most legible. Also, string.format is always better in LuaJIT for performance reasons. Try something like this:

```lua
string.format([[<!DOCTYPE html>
<html>
<head><title>%1$s</title></head>
<body>
<h1>%1$s</h1>
%2$s
</body>
</html>]], feed.title, feed.description)
```